app/views/emails/mediaItem.php

```diff
@@ -791,7 +791,7 @@
                         <table class="twelve columns">
                           <tr>
                             <td>
-                              <center><img class="center" height="50" style="height: 50px !important;" src="<?=asset("assets/img/logo.png");?>"></center>
+                              <center><img class="center" height="50" style="height: 50px !important;" src="<?=asset("assets/img/logo.svg");?>"></center>
```

Comment:
It's probably safer to provide a png for emails. I'm not sure if all mail clients can do svgs.

Reply:
Bloody 'ell, what year is this?! True, though.